README.md

```diff
@@ -107,11 +107,11 @@ You can also force all defined rounds to be open by setting the environment vari
 To seed fund & round data to db
 
 ```
-docker exec -ti <fund_store_container_id> scripts/load_cof_r2.py
+docker exec -ti <fund_store_container_id> python -m scripts.load_cof_r2
```

Comment:
```
/usr/local/bin/python: No module named scripts/load_cof_r3w1
```

```suggestion
docker exec -ti <fund_store_container_id> python scripts.load_cof_r2.py
```

Reply:
My bad.  Pulled your branch and it works

Reply:
No worries. Thanks for the approval :)